specification/dimensions/usageunit.md

```diff
@@ -33,18 +43,37 @@ Units listed in the following table are presented in the following format: "abbr
 
 If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning or incompatible capitalization.  An example of unit now allowed is a unit such as "calls" or "number" to signify the number of times an event occurred in this case, the unit "count" MUST be used to maintain compatibility with this specification.
 
-| TIME_UNIT        | DATA_UNIT | COUNT_UNIT   | NUMERIC PREFIXES        |
-|------------------|-----------|--------------|-------------------------|
-| yr: year         | bit: bit  | cnt: count   | Ki: kibi (2^10)         |
-| mo: month        | B: byte   |              | Mi: mebi (2^20)         |
-| wk: week         |           |              | Gi: gibi (2^30)         |
-| d: day           |           |              | Ti: tebi (2^40)         |
-| h: hour          |           |              | Pi: pebi (2^50)         |
-| min: minute      |           |              | K: kilo (10^3)          |
-| s: second        |           |              | M: mega (10^6)          |
-| ms: millisecond  |           |              | G: giga (10^9)          |
-| us: microsecond  |           |              | T: tera (10^12)         |
-| ns: nanosecond   |           |              | P: peta (10^15)         |
+The following table lists the valid units for 3 dimensions of measurement: time, data, and count.  
+
+| Time             | Data      | Count        |
+|------------------|-----------|--------------|
+| yr: year         | bit: bit  | cnt: count   |
```

Comment:
Can we use "Units" as the generic count? It sounds better and aligns more with the other values. 
"24 Hours" 
"15 Units"

Also, why are we not allowing other common words? I'm fine if we don't have everything, but there are a core set of common words we should be able to use.

FYI - Here are Microsoft's unique values: https://github.com/microsoft/finops-toolkit/blob/dev/src/open-data/PricingUnits.csv
```suggestion
| yr: year         | bit: bit  | Units   |
```

Reply:
The only problem is that multiple of the vendors use the generic "count".

Reply:
@JGrubb @flanakin @amitw-g - I've gone back and forth here, but I'm aligned with what Michael's saying here, why would we not just allow a provider to use a unit term like "Token", "Domain", "Certificate"?  There is a large list here.

What I ended up proposing is this "Use the Allowed Values, but if you have a term for count-based unit that isn't present.  Anything unrecognized will be considered a new count-based unit."     I really don't want to get a charge from a provider for 400000 count on a Gen AI service, I'd like to know exactly what the count unit was.